### ISSUE TITLE
the parameter setting interface of PyRAF does not pop up(Apple M2 Pro)

### ISSUE THREAD

**Issue by yuanqastro1996** [iraf-community/pyraf#152]
**Describe the bug**
I successfully installed IRAF(2.17.1), X11raf. When I use the Pyraf (2.2.1), the parameter setting interface does not pop up; for example, when I type noao/imred/ccdred/epar zeroxombine, it gets stuck. When I close the termination window, it shows that ''closing this window will terminate the running process: zsh, python, x_system.e.'' How should I solve it? Looking forward to your help. I used 'pip install pyraf (pip3 install pyraf)' to install Pyraf.

**To Reproduce**

type cl show:  
```
  Community IRAF V2.17.1        

  This product includes results achieved by the IRAF64 project in 2006-
  2009 directed by Chisato Yamauchi (C-SODA/ISAS/JAXA).

  Welcome to IRAF.  To list the available commands, type ? or ??.  To get
  detailed information about a command, type `help <command>'.  To run  a
  command  or  load  a  package,  type  its name.   Type  `bye' to exit a
  package, or `logout' to get out  of the CL.

  Visit http://github.com/iraf-community/iraf/issues to report problems.

  ***  Using global login file:  /Users/yuanq/.iraf/login.cl
  The following commands or packages are currently defined:

      dataio.     language.   noao.       plot.       softools.   utilities.
      images.     lists.      obsolete.   proto.      system.
cl> 
```

type pyraf show:
```
(base) yuanq@YUANQDEMACBOOK-PRO ~ % pyraf
setting terminal type to 'xgterm' ...

  Community IRAF V2.17.1

  This product includes results achieved by the IRAF64 project in 2006-
  2009 directed by Chisato Yamauchi (C-SODA/ISAS/JAXA).

  Welcome to IRAF.  To list the available commands, type ? or ??.  To get
  detailed information about a command, type `help <command>'.  To run  a
  command  or  load  a  package,  type  its name.   Type  `bye' to exit a
  package, or `logout' to get out  of the CL.

  Visit http://github.com/iraf-community/iraf/issues to report problems.

  ***  Using global login file:  /Users/yuanq/.iraf/login.cl
  The following commands or packages are currently defined:

clpackage/:
 clpackage/     language/       obsolete/       softools/       utilities/
 dataio/        lists/          plot/           system/
 images/        noao/           proto/          user/
PyRAF 2.2.1
Python/CL command line wrapper
  .help describes executive commands
-->
```

pip install pyraf show:
```
DEPRECATION: Loading egg at /Users/yuanq/anaconda3/lib/python3.11/site-packages/PyZOGY-0.0.1-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at /Users/yuanq/anaconda3/lib/python3.11/site-packages/fonttools-4.41.1-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
Requirement already satisfied: pyraf in ./anaconda3/lib/python3.11/site-packages (2.2.1)
Requirement already satisfied: numpy in ./anaconda3/lib/python3.11/site-packages (from pyraf) (1.24.3)
Requirement already satisfied: configobj in ./anaconda3/lib/python3.11/site-packages (from pyraf) (5.0.8)
Requirement already satisfied: six in ./anaconda3/lib/python3.11/site-packages (from configobj->pyraf) (1.16.0)
```

**Expected behavior**
I want when I use pyraf, it pops up the parameter setting interface.  

**System details:**
 - OS: [e.g. macOS 13.6 (22G120), Apple M2 Pro]
 - Version [e.g. 2.17.1]
 - Versions of all additionally involved software, i.e. xgterm(by x11 install), PyRAF(2.2.1).

**Comment by olebole** [comment]
I transferred the issue to PyRAF, since this is a problem there. I think that in the macOS Ventura, the Python Tk packge is buggy. There is another problem described in [iraf-community#309](https://github.com/orgs/iraf-community/discussions/309).

I don't have access to a recent Mac for the moment, so I can't fix this yet. On a recent Debian, and an old macOS (HighSierra) I couldn't reproduce the issue.

**Comment by olebole** [comment]
This is probably already fixed in #145, although we didn't make a new release yet.
